<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
  * Released as open source by Gabriel Caudrelier
  *
  * Developed by Gabriel Caudrelier, gabriel dot caudrelier at gmail dot com
  *
  * https://github.com/metrodango/pip3line
  *
  * Released under BSD 3-Clause see LICENSE for more information
</comment>
 <class>TimestampWidget</class>
 <widget class="QWidget" name="TimestampWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>249</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignTop">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="formatLabel">
        <property name="text">
         <string>Time String format:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="formatLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Default timezone when parsing a string</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="tzLocalRadioButton">
        <property name="text">
         <string>Local Time Zone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="tzUTCRadioButton">
        <property name="text">
         <string>UTC</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
